### PR TITLE
Created the ability to run targets from a project directory

### PR DESCRIPTION
**Pull request by trentis**
## Description

Please describe the aim of the pull request and the changes made in the commits.
Changes in run.go and main.go. To provided the capability to run virtual machines using targets from a directory

```
vorteil run helloworld/:run
vorteil run ..\..\helloworld\:run
vorteil run C:\Users\Documents\helloworld:run
```

## Purpose

- [ ] Bug fix
- [x] New feature
- [ ] Other

## How was this tested? (if applicable)

Tested on windows and using wsl for linux,

## Checklist

- [x] Code is commented
- [x] PR is signed

### CODE REVIEW

File: cmd/vorteil/main.go [hunk 1]
```diff
@@ -121,6 +122,38 @@ const (
...
+	var target string
+
+	// colonSplit will always be 1 when strings.splitting something that doesn't have it
+	if len(colonSplit) > 1 {
+		src = colonSplit[0]
+		target = colonSplit[1]
+
+		// Windows has one extra coloumn due to C:\ from abs
+		if runtime.GOOS == "windows" {
+			src = fmt.Sprintf("%s%s", "C:", colonSplit[1])
```
Comment (jensg-st) [inline cmd/vorteil/main.go:143]: what happens if I have disk D:?

File: cmd/vorteil/main.go [hunk 2]
```diff
@@ -121,6 +122,38 @@ const (
...
+	if err != nil {
+		return "", "", err
+	}
+
+	// Check if source contains a target
+	colonSplit := strings.Split(src, ":")
+	var target string
+
+	// colonSplit will always be 1 when strings.splitting something that doesn't have it
+	if len(colonSplit) > 1 {
```
Comment (jensg-st) [inline cmd/vorteil/main.go:137]: maybe check with LastIndex and cut string there? It is either "not found" or something legit.

**Review by jensg-st — CHANGES_REQUESTED** [review]

File: cmd/vorteil/main.go (hunk 1 above)
Comment (trentis) [reply cmd/vorteil/main.go]: I ended up using the colonSplit variable properly to replace it with whatever letter gets split

File: cmd/vorteil/main.go (hunk 2 above)
Comment (trentis) [reply cmd/vorteil/main.go]: I'm now setting target to the last of the split array. A few other if statments exist like

Set target empty if its the same as src. Happens on linux
```
if target == src {
 target = ""
}
```
And the following for windows. If windows doesn't have a length of 3 then the target is not provided 
```
if colonLength == 2 {
target = ""
}
```

**Review by jensg-st — APPROVED** [review]

**Pull request merged**